docs/modules/usage/how-to/custom-sandbox-guide.md

```diff
@@ -16,17 +16,23 @@ If you choose the first option, you can skip the `Create Your Docker Image` sect
 
 ## Create Your Docker Image
 
-To create a custom Docker image, it must be Debian based.
+To create a custom Docker image, you can use any base image that supports your requirements. While Debian-based images are common, they're not strictly required.
```

Comment:
Did you try this with a non-debian based image? I tried it once for a simple ruby image and it did not work. Maybe it was fixed.

Reply:
I'll revert this...

```suggestion
To create a custom Docker image, it must be Debian based.
```